views/Graph.react.js

```diff
@@ -390,54 +483,94 @@ class Graph extends Component {
     socket.close()
     window.removeEventListener('resize', this.handleResize)
   }
-
+  updateUserPreferences = (configuration) => {
```

Comment:
is this updating a preference or creating a new one?

Reply:
https://github.com/AMP-SCZ/dpdash/blob/186e1697f1e18e5db2e4e30d67dd6e6eaaca79ce/server/routes/index.js#L853

They went with a post when it should have been a PUT. Definitely an opportunity to refactor a few things. What do you think, should I refactor it on this ticket? or make tickets of these opportunities?

Reply:
gotcha, so it is updating not creating a new one. I think we can save that refactor for another time, but we may want to rename the `createNewPreference` variable in this function so it doesn't suggest that we're creating a new one.

Reply:
copy that 
https://github.com/AMP-SCZ/dpdash/pull/129/commits/5f670ec6f3a88da23fbfda4601cab2b7bcec7705

Reply:
I believe this configuration is currently the ID of a configuration (unless we update the form to call this function with the configuration object).

Reply:
Yes the configuration is the id. I can rename it to say configurationId, because the other pieces are updated to work with just the ID and not the entire configuration object.